chef-windows.gemspec

```diff
@@ -0,0 +1,22 @@
+gemspec = eval(IO.read(File.expand_path("../chef.gemspec", __FILE__)))
+
+gemspec.platform = Gem::Platform.new ["universal", "mingw32", nil]
```

Comment:
Oh yay!  I like the extra parens to make function calls explicit but that supposedly offends the ruby sensibility of some others.  I'm quite happy to put it back in.